packages/react-components/react-toast/src/state/useToastFactory.ts

```diff
@@ -0,0 +1,21 @@
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import { createToast as createToastVanilla } from './vanilla/createToast';
+import * as React from 'react';
+import { ToastOptions } from './types';
+
+export function useToastFactory() {
```

Comment:
I'd call this `useToastDispatcher`, following up with comments on the `createToast` method

Reply:
There will be other methods from this like clearing/pausing toasts. What about `useToastController`